MatterControlLib/Utilities/WebUtilities/WebCache.cs

```diff
@@ -244,8 +241,7 @@ public static void RetrieveText(string uriToLoad, Action<string> updateResult)
 				{
 					try
 					{
-						textFileName = AggContext.StaticData.ReadAllText(Path.Combine("TextWebCache", longHash.ToString() + ".txt"));
-						fileText = File.ReadAllText(textFileName);
+						fileText = AggContext.StaticData.ReadAllText(Path.Combine("TextWebCache", longHash.ToString() + ".txt"));
```

Comment:
Looks like this never worked and wasn't hit often enough to pick up. The second call to ReadAllText tries to use the markdown text returned from the first call as the file path, always throwing

Reply:
Thanks for the catch and the fix.